index.html

```diff
@@ -0,0 +1,1595 @@
+                      <h5 class="card-title text-danger text-uppercase m-0 bg-light">U.S</h5>
+                    </div>
+                  </div>
+                  <div class="card-body p-0 mt-1">
+                    <h5 class="card-title font-weight-bold h4">'Educated People' in DHS Know Climate Change Is Driving
+                      Migration: Expert</h5>
+                    <p class="card-text">"Some might be hoping...we'll go back to a more enlightened moment in the
+                      history of the country," said Oscar Chacón, co‐founder and executive director of Alianza Americas.
+                    </p>
+                  </div>
+                </div>
+
+
+                <div class="card text-body border-0 pb-4">
+                  <div class="wrap position-relative">
+                    <img
+                      src="https://d.newsweek.com/en/full/1529454/candidates-attend-congressional-hispanic-caucus-institutes-presidential-forum.jpg?w=397&h=265&q=90&f=6ccbace76f3116da7433888695a31146"
+                      class="card-img" alt="...">
+                    <div class="card-img-overlay p-0 position-absolute fixed-bottom ">
+                      <h5 class="card-title text-danger text-uppercase m-0 bg-light">Politics</h5>
+                    </div>
+                  </div>
+
+                  <div class="card-body p-0 mt-1">
+                    <h5 class="card-title font-weight-bold h4">Cash-Strapped Booker Campaign Faces Dire Straits</h5>
+                    <p class="card-text">The campaign manager himself acknowledged the existential challenge the
+                      campaign's shaky finances present to their chances of success.</p>
+                  </div>
+                </div>
+
+                <div class="card text-body border-0 pb-4">
+                  <div class="wrap position-relative">
+                    <img
+                      src="https://d.newsweek.com/en/full/1528794/james-cameron.jpg?w=397&h=265&q=90&f=b74aa61b116929666a48b4bc6d2738b3"
+                      class="card-img" alt="...">
+                    <div class="card-img-overlay p-0 position-absolute fixed-bottom">
+                      <h5 class="card-title text-danger text-uppercase m-0 bg-light">Tech & Science</h5>
+                    </div>
+                  </div>
+                  <div class="card-body p-0 mt-1">
+                    <h5 class="card-title font-weight-bold h4">James Cameron: Our So-Called Civilization Is Using the
+                      Ocean as Its Toilet</h5>
+                    <p class="card-text">The director said deep sea research is "shamefully underfunded" and called for
+                      a "global fleet of swarm robotics" to study Earth's oceans. </p>
+                  </div>
+                </div>
+
+              </div>
+
+              <!-- TOP Story -->
+              <div class="col-lg-7 col-md-6 order-1 order-md-2 position-relative overflow-auto" id="top-story">
```

Comment:
Please use ```<article>``` tags for all sections containing articles.

Reply:
Done